PlexRequests.Core/SettingModels/PlexRequestSettings.cs

```diff
@@ -58,7 +58,8 @@ public PlexRequestSettings()
         public bool Wizard { get; set; }
         public bool DisableTvRequestsByEpisode { get; set; }
         public bool DisableTvRequestsBySeason { get; set; }
-
+        //To Remove at some point? 
+        public bool SendRecentlyAddedEmail { get; set; }
```

Comment:
I usually apply the `[Obsolete]` attribute with a description. It add's build warnings which in a few versions time I can then remove :)